Mathlib/Order/Copy.lean

```diff
@@ -20,6 +20,20 @@ universe u
 
 variable {α : Type u}
 
+/-- A function to create a provable equal copy of a top order
```

Comment:
Can the changes to this file be cherry-picked?

Reply:
#15006